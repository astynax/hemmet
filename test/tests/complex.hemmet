form:search-form$theme>input.query^red-text>(div.help~hidden_t)+span.submit&button~text_small>.hint
