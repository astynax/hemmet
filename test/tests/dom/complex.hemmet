#container>.nav>(ul.menu>li.item+li.item)+#content.width-800.selected>{text with space after }+{ text with space before}
