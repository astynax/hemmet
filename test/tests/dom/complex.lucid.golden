div_ [id_ "container"] $ do
  div_ [class_ "nav"] $ do
    ul_ [class_ "menu"] $ do
      li_ [class_ "item"]
      li_ [class_ "item"]
  div_ [id_ "content", classes_ ["width-800", "selected"]] $ do
    "text with space after "
    " text with space before"
